build_scripts/elk_setup/elk_setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Install Java
+yum -y install java-1.8.0-openjdk
+
+# Download and install Elasticsearch
+wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.2.rpm -P /tmp
+rpm -ivh /tmp/elasticsearch-5.6.2.rpm
+chkconfig --add elasticsearch
+
+# Configure Elasticsearch
+mv /etc/elasticsearch/elasticsearch.yml /etc/elasticsearch/elasticsearch.yml.orig
+ln -s /vagrant/build_scripts/elk_setup/elasticsearch.yml /etc/elasticsearch/elasticsearch.yml
+mkdir -p /data/es_logs
+chown -R vagrant:vagrant /data
+chmod 777 -R /data
+# Add the following lines to /etc/security/limits.conf for max number of threads error
```

Comment:
Can you use echo to write these lines to `/etc/security/limits.conf`?

Something like
```bash
# Increase the maximum number of threads ElasticSearch is allowed to use
cat << 'EOF' >> /etc/security/limits.conf
elasticsearch - nproc 2048
elasticsearch - nofile 65536
EOF
```